sentry-android-core/src/main/java/io/sentry/android/core/performance/AppStartMetrics.java

```diff
@@ -137,12 +146,27 @@ public long getClassLoadedUptimeMs() {
       // Only started when sdk version is >= N
       final @NotNull TimeSpan appStartSpan = getAppStartTimeSpan();
       if (appStartSpan.hasStarted()) {
-        return appStartSpan;
+        return validateAppStartSpan(appStartSpan);
       }
     }
 
     // fallback: use sdk init time span, as it will always have a start time set
-    return getSdkInitTimeSpan();
+    return validateAppStartSpan(getSdkInitTimeSpan());
+  }
+
+  private @NotNull TimeSpan validateAppStartSpan(final @NotNull TimeSpan appStartSpan) {
+    long spanStartMillis = appStartSpan.getStartTimestampMs();
+    long spanEndMillis =
+        appStartSpan.hasStopped()
+            ? appStartSpan.getProjectedStopTimestampMs()
+            : SystemClock.uptimeMillis();
```

Comment:
`h`: span start/stop `TimestampMs` are in unix time, so this should be unix time as well right?
```suggestion
            : System.currentTimeMillis();
```

Reply:
yep, nice catch 😄